src/TouristInfoCards.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+const TouristInfoCards = () => {
+  return (
+    <div className="card-container">
+      <div className="card">
+        <img
+          src="https://peoplemakeglasgow.com/imager/general/548549/European-Sport-Capital_Header-Image-1_961ee30b631a75b37d23eb0e09df30ea.png"
+          className="card-img-top"
+          target="_blank"
+        />
+        <div className="card-body">
+          <a
+            href="https://peoplemakeglasgow.com"
+            className="btn btn-primary"
+            target="_blank"
+          >
+            Go somewhere
+          </a>
+        </div>
+      </div>
+      <div className="card">
+        <img
+          src="https://eu-assets.simpleview-europe.com/manchester2016/imageresizer/?image=%2Fdbimgs%2FManOne-03%284%29.jpg&action=Background_Overlay"
+          className="card-img-top"
+        />
+        <div className="card-body">
+          <a
+            href="https://peoplemakeglasgow.com"
+            className="btn btn-primary"
+            target="_blank"
+          >
+            Go somewhere
+          </a>
+        </div>
+      </div>
+      <div className="card">
+        <img
+          src="https://peoplemakeglasgow.com/imager/general/548549/European-Sport-Capital_Header-Image-1_961ee30b631a75b37d23eb0e09df30ea.png"
+          className="card-img-top"
+          target="_blank"
+        />
+        <div className="card-body">
+          <a
+            href="https://peoplemakeglasgow.com"
+            className="btn btn-primary"
+            target="_blank"
+          >
+            Go somewhere
+          </a>
+        </div>
+      </div>
+    </div>
+  );
```

Comment:
Not part of the homework, but theres an improvement that could be made to this file to avoid duplication, can you see it?

Reply:
Thanks Sam. I am not sure, but should I create an array of objects from the src(s) and pass them to a new function called creatCard( ) ?

Reply:
I was jsut meaning you could create a new component called InfoCard and pass some props to it :)